docs-source/content/faq/domain-secret-mismatch.md

```diff
@@ -54,9 +54,9 @@ may have different images under the same tag in their individual, local Docker r
 The simplest solution is to set `imagePullPolicy` to `Always`; however, the better solution would be to design your development
 pipeline to generate new Docker image tags on every build and to never reuse an existing tag.
 
-### Domain on a persistent volume
+#### Domain on a persistent volume
 
-#### 1. Completely replacing the domain directory
+##### 1. Completely replacing the domain directory
```

Comment:
It looks a little odd that this a numbered list, since the list contains just one entry.

Reply:
Good point. Maybe each entry, this one and the two above it, can be bulleted items. I'll confirm with @rjeberhard